### ISSUE TITLE
Pressing F7 does nothing, config file is never created [1.18.2 Fabric]

### ISSUE THREAD

**Issue by Nataly171** [shedaniel/LightOverlay-Fabric#152]
So I've had this mod in my pack for quite some time now but it's never worked. Pressing the F7 key simply doesn't do anything. I tried looking in the config, but there is no config file for this mod to be seen. I might be missing something?
[latest.log](https://github.com/shedaniel/LightOverlay/files/9313698/latest.log)

**Comment by Mourdraug** [comment]
Same thing here, it shows up in mod menu but without a config button Only on 1.18.2 tho, 1.19 ver works fine 
[latest.log](https://github.com/shedaniel/LightOverlay/files/9592586/latest.log)

**Comment by Leosky** [comment]
Same issue on 6.0.5, i downgraded to 6.0.4 and it seems to create the files but the game crash at startup.

**Comment by Bread-Ch4n** [comment]
I downloaded the mod from the site and not through the curseforge app and it works now.

**Comment by Bread-Ch4n** [comment]
Cause when I looked at the mod name in the app it said the mod was 6.1.0, not 6.0.5

**Comment by Bread-Ch4n** [comment]
Also check if u aren't using the Architectury API (UNUSED).
Use this one: https://www.curseforge.com/minecraft/mc-mods/architectury-api

**Comment by suduko666** [comment]
yes for some reason the curseforge app has a 6.1.0 fabric  the full name for the version and it defaults to that to install as its marked as a release and the 6.0.5 as beta and after you have to manually delete the 6.1.0 as the app wont even if you uninstall it in the app, if you make sure the install the .5 manually in the app before the 6.1 is installed or after you make sure its deleted everything works fine
